<!doctype html>
<!--
  Minimal Mistakes Jekyll Theme 4.8.1 by Michael Rose
  Copyright 2017 Michael Rose - mademistakes.com | @mmistakes
  Free for personal and commercial use under the MIT license
  https://github.com/mmistakes/minimal-mistakes/blob/master/LICENSE.txt
-->
<html lang="en" class="no-js">
  <head>
    <meta charset="utf-8">

    


<!-- begin SEO -->
<title>Publications - Chris Kreucher</title>
<meta name="description" content="Chris Kreucher's Publications">
<meta name="author" content="Chris Kreucher">
<meta property="og:locale" content="en">
<meta property="og:site_name" content="Chris Kreucher">
<meta property="og:title" content="Publications">

<link rel="canonical" href="https://chriskreucher.github.io//publications/publications.html/">
<meta property="og:url" content="https://chriskreucher.github.io//publications/publications.html/">
<!-- end SEO -->



<link href="https://www.amirasiaee.com/feed.xml" type="application/atom+xml" rel="alternate" title="Amir Asiaee Feed">

<!-- http://t.co/dKP3o1e -->
<meta name="HandheldFriendly" content="True">
<meta name="MobileOptimized" content="320">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<script>
  document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + ' js ';
</script>

<!-- Load jQuery -->
<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
<!-- Load KaTeX -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-beta/katex.min.css" integrity="sha384-L/SNYu0HM7XECWBeshTGLluQO9uVI1tvkCtunuoUbCHHoTH76cDyXty69Bb9I0qZ" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.9.0-beta/katex.min.js" integrity="sha384-ad+n9lzhJjYgO67lARKETJH6WuQVDDlRfj81AJJSswMyMkXTD49wBj5EP004WOY6" crossorigin="anonymous"></script>

<!-- For all browsers -->
<link rel="stylesheet" href="https://www.amirasiaee.com/assets/css/main.css">

<!--[if lte IE 9]>
  <style>
    /* old IE unsupported flexbox fixes */
    .greedy-nav .site-title {
      padding-right: 3em;
    }
    .greedy-nav button {
      position: absolute;
      top: 0;
      right: 0;
      height: 100%;
    }
  </style>
<![endif]-->


    <!-- start custom head snippets -->

<!-- insert favicons. use http://realfavicongenerator.net/ -->

<!-- end custom head snippets -->
  </head>

  <body class="layout--single">

    <!--[if lt IE 9]>
<div class="notice--danger align-center" style="margin: 0;">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience.</div>
<![endif]-->

    <div class="masthead">
  <div class="masthead__inner-wrap">
    <div class="masthead__menu">
      <nav id="site-nav" class="greedy-nav">
        <a class="site-title" href="https://www.amirasiaee.com/">Amir Asiaee</a>
        <ul class="visible-links">
          
            
            <li class="masthead__menu-item">
              <a href="https://www.amirasiaee.com/bio" >Bio</a>
            </li>
          
            
            <li class="masthead__menu-item">
              <a href="https://www.amirasiaee.com/publicationbyyear" >Publication</a>
            </li>
          
            
            <li class="masthead__menu-item">
              <a href="https://www.amirasiaee.com/research" >Research</a>
            </li>
          
            
            <li class="masthead__menu-item">
              <a href="https://www.amirasiaee.com/teaching" >Teaching</a>
            </li>
          
            
            <li class="masthead__menu-item">
              <a href="https://www.amirasiaee.com/blog" >Blog</a>
            </li>
          
            
            <li class="masthead__menu-item">
              <a href="https://www.amirasiaee.com/joinus" >Join Us</a>
            </li>
          
        </ul>
        
        <button class="search__toggle" type="button">
          <svg class="icon" width="16" height="16" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 15.99 16">
            <path d="M15.5,13.12L13.19,10.8a1.69,1.69,0,0,0-1.28-.55l-0.06-.06A6.5,6.5,0,0,0,5.77,0,6.5,6.5,0,0,0,2.46,11.59a6.47,6.47,0,0,0,7.74.26l0.05,0.05a1.65,1.65,0,0,0,.5,1.24l2.38,2.38A1.68,1.68,0,0,0,15.5,13.12ZM6.4,2A4.41,4.41,0,1,1,2,6.4,4.43,4.43,0,0,1,6.4,2Z" transform="translate(-.01)"></path>
          </svg>
        </button>
        
        <button class="greedy-nav__toggle" type="button">
          <span class="visually-hidden">Toggle Menu</span>
          <div class="navicon"></div>
        </button>
        <ul class="hidden-links hidden"></ul>
      </nav>
    </div>
  </div>
</div>

    <div class="initial-content">
      



<div id="main" role="main">
  
  <div class="sidebar sticky">
  

<div itemscope itemtype="http://schema.org/Person">

  
    <div class="author__avatar">
      

      
        <img src="https://www.amirasiaee.com/assets/images/shot.jpg" alt="Amir Asiaee" itemprop="image">
      
    </div>
  

  <div class="author__content">
    
      <h3 class="author__name" itemprop="name">Amir Asiaee</h3>
    
    
      <p class="author__bio" itemprop="description">
        Machine learning researcher dabbling in genomics and cancer study.
      </p>
    
  </div>

  <div class="author__urls-wrapper">
    <button class="btn btn--inverse">Follow</button>
    <ul class="author__urls social-icons">
      

      

      
        <li>
          <a href="mailto:amir.asiaeetaheri@vumc.org">
            <meta itemprop="email" content="amir.asiaeetaheri@vumc.org" />
            <i class="fa fa-fw fa-envelope-square" aria-hidden="true"></i> Email
          </a>
        </li>
      

      

      

      

      

      
        <li>
          <a href="https://www.linkedin.com/in/amir-asiaee-28478532" itemprop="sameAs">
            <i class="fa fa-fw fa-linkedin-square" aria-hidden="true"></i> LinkedIn
          </a>
        </li>
      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      

      <!--
  <li>
    <a href="http://link-to-whatever-social-network.com/user/" itemprop="sameAs">
      <i class="fa fa-fw" aria-hidden="true"></i> Custom Social Profile Link
    </a>
  </li>
-->
    </ul>
  </div>
</div>

  
  </div>


  <article class="page" itemscope itemtype="http://schema.org/CreativeWork">
    <meta itemprop="headline" content="Publication">
    
    
    

    <div class="page__inner-wrap">
      
        <header>
          <h1 class="page__title" itemprop="headline">Publication
</h1>
          
        </header>
      

      <section class="page__content" itemprop="text">
        
          <aside class="sidebar__right">
            <nav class="toc">
              <header><h4 class="nav__title"><i class="fa fa-file-text"></i> On This Page</h4></header>
              <ul class="toc__menu">
  <li><a href="#">In the Pipeline</a></li>
  <li><a href="#2020">2020</a></li>
  <li><a href="#2019">2019</a></li>
  <li><a href="#2018">2018</a></li>
  <li><a href="#2016">2016</a></li>
  <li><a href="#2015">2015</a></li>
  <li><a href="#2013">2013</a></li>
  <li><a href="#2012">2012</a></li>
</ul>
            </nav>
          </aside>
        
        <p> 
You can also find my articles on my 
<a href= "https://scholar.google.com/citations?hl=en&user=RrV5LtsAAAAJ&view_op=list_works&gmla=AJsN-F6BbctU_y2tmfyng0h6cWnHrLFzWiFSvoR02lweIIDZGFcuBB245xJ8JefosLrS96ffCDwCIkdPlZn-8eQDIvzxHhYrDN3EWPVNKpkCbMGRVUvtNtKR58b7tNfZwzIlzGLSrW9uXAB1jwgdzRRd28xH-OvIBptqCNyMWIEls2cxAXrElio&gmla=AJsN-F7ExyPlRXeGaL84c79D8ZhMEAsp07i-ltd5qTpqpi1tJkM76MaCG39GO8wwT0ZqYQH5DNoRDgHc800LcH3PHzyfDaeSJLKw89q2eEVie4MXx4EpKFY&sciund=1600418140394454477&gmla=AJsN-F5qzBXJC9vt5YlwCiefO0cO9I5luN-t5SmD6JapDVtq2sRzeaizO17S2hsWqtDtiBkX-4fNqzKxD36whzjqhqllQhmDqzX2t_rB8jMzft8S1TPgAKs&gmla=AJsN-F75HUAZE7OFVfC73zIqJ6r5XfBHkuhMS-Ha05DOLp__GaBrNe7KrF9XXS_0JXuR0dyH_aDMSaWsF4RM82MStQdjdQKFZ20KfQETpyIlV9Lk6Ye3cb8"> Google Scholar profile</a>.  
</p> 
<p>
* Indicates equal contribution.
</p> 
<a href="/publicationbytopic/"><button type="button" class="btn" style="outline:none">By Topic </button></a> 
<a href="/publicationbyyear/"><button type="button" class="btn" style="outline:none">By Year  </button></a> 

<h3  class="pubyear">In the Pipeline</h3>
<ol class="bibliography"><li><span id="aans19a"><b>Asiaee*, A.</b>, Abrams*, Z. B., Nakayiza, S., Sampath, D., &amp; Coombes, K. R. (2019). <i>Identification and comparison of genes differentially regulated
              by transcription factors and miRNAs</i>.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_aans19a-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_aans19a-bibtex" style="outline:none;">BibTeX</button>
    
    
	
	
	
	
	<a href="https://doi.org/10.1101/803643"><button class="btn btnId btnPub--supplement" style="outline:none; position:relative;white-space: normal;">URL</button></a>
    

	<div class="dropDownBibtex" id="aans19a-bibtex">
    <pre>@article{aans19a,
  title = {Identification and comparison of genes differentially regulated
                by transcription factors and {miRNAs}},
  author = {Asiaee*, Amir and Abrams*, Zachary B and Nakayiza, Samantha and Sampath, Deepa and Coombes, Kevin R},
  archiveprefix = {bioRxiv},
  eprint = {803643},
  note = {https://doi.org/10.1101/803643},
  year = {2019}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="aans19a-abstract">
    Transcription factors and microRNAs (miRNA) both play a critical role in gene regulation and in the development of many diseases such as cancer. Understanding how transcription factors and miRNAs influence gene expression is thus important to understand, but complicated due to the large and interconnected nature of the human genome. To help better understand what genes are being regulated by transcription factors and/or miRNAs we looked at it over 8000 patient samples from 32 different cancer types collected from The Cancer Genome Atlas (TCGA). We started by clustering the transcription factors and miRNAs using Thresher to reduce the number of features. Using both the mRNA and miRNA sequencing data we constructed linear models to calculate the coefficient of determination (R2) for each mRNA based on the Thresher cluster expression. We generated three types of linear models: transcription factor, miRNA and transcription factor plus miRNA. We then determined genes that were highly explained or poorly explained by each of the three models based on the genes R2 value. We performed downstream gene enrichment analysis using ToppGene on the sets of well and poorly explained genes. This identified differences in gene regulation between transcription factors and miRNAs and showed what groups of gene are differentially regulated.
    </div>
	
</li>
<li><span id="aobc18"><b>Asiaee, A.</b>, Oymak, S., Coombes, K. R., &amp; Banerjee, A. High Dimensional Data Enrichment: Interpretable, Fast, and
                   Data-Efficient. <i>Under Review In</i>.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_aobc18-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_aobc18-bibtex" style="outline:none;">BibTeX</button>
    
    
	
	
	
	
	<a href="https://arxiv.org/abs/1806.04047"><button class="btn btnId btnPub--supplement" style="outline:none; position:relative;white-space: normal;">URL</button></a>
    

	<div class="dropDownBibtex" id="aobc18-bibtex">
    <pre>@article{aobc18,
  title = {High Dimensional Data Enrichment: Interpretable, Fast, and
                     {Data-Efficient}},
  author = {Asiaee, Amir and Oymak, Samet and Coombes, Kevin R and Banerjee, Arindam},
  journal = {Under review in},
  archiveprefix = {arXiv},
  primaryclass = {stat.ML},
  eprint = {1806.04047},
  note = {https://arxiv.org/abs/1806.04047}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="aobc18-abstract">
    High dimensional structured data enriched model describes groups of observations by shared and per-group individual parameters, each with its own structure such as sparsity or group sparsity. In this paper, we consider the general form of data enrichment where data comes in a fixed but arbitrary number of groups G.  Any convex function, e.g., norms, can characterize the structure of both shared and individual parameters. We propose an estimator for high dimensional data enriched model and provide conditions under which it consistently estimates both shared and individual parameters. We also delineate sample complexity of the estimator and present high probability non-asymptotic bound on estimation error of all parameters. Interestingly the sample complexity of our estimator translates to conditions on both per-group sample sizes and the total number of samples. We propose an iterative estimation algorithm with linear convergence rate and supplement our theoretical analysis with synthetic and real experimental results. Particularly, we show the predictive power of data enriched model along with its interpretable results in anticancer drug sensitivity analysis.
    </div>
	
</li>
<li><span id="ajgs20">Abrams, Z. B., Joglekar, A., Gershkowitz, G. R., Sinicropiyao, S., <b>Asiaee, A.</b>, Carbone, D. P., &amp; Coombes, K. R. Personalized Transcriptomics: Selecting Drugs Based on Gene Expression Profiles. <i>Under Review</i>.</span>


    

	<button class="btn btnId btnPub--BibTex" id="b_ajgs20-bibtex" style="outline:none;">BibTeX</button>
    
    
	
	
	
	

	<div class="dropDownBibtex" id="ajgs20-bibtex">
    <pre>@article{ajgs20,
  title = {Personalized Transcriptomics: Selecting Drugs Based on Gene Expression Profiles},
  author = {Abrams, Zachary B. and Joglekar, Anoushka and Gershkowitz, Gregory R. and Sinicropiyao, Sara and Asiaee, Amir and Carbone, David P. and Coombes, Kevin R.},
  journal = {Under review}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="ajgs20-abstract">
    
    </div>
	
</li></ol>


  <h3  id="2020" class="pubyear">2020</h3>
  <ol class="bibliography"><li><span id="aans20"><b>Asiaee*, A.</b>, Abrams*, Z. B., Nakayiza, S., Sampath, D., &amp; Coombes, K. R. (2020). Explaining Gene Expression Using Twenty-One MicroRNAs. <i>Journal of Computational Biology</i>, <i>Forthcoming</i>.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_aans20-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_aans20-bibtex" style="outline:none;">BibTeX</button>
    
    
	
	
	
	

	<div class="dropDownBibtex" id="aans20-bibtex">
    <pre>@article{aans20,
  title = {Explaining Gene Expression Using Twenty-One MicroRNAs},
  author = {Asiaee*, Amir and Abrams*, Zachary B. and Nakayiza, Samantha and Sampath, Deepa and Coombes, Kevin R.},
  journal = {Journal of computational biology},
  volume = {Forthcoming},
  year = {2020}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="aans20-abstract">
    The transcriptome, or gene expression profile, of a tumor contains detailed information about the disease. Although advances in sequencing tech-nologies have generated larger and more infor-mative data sets, there are still many questions about how the transcriptome is regulated. One class of regulatory elements consists of micro-RNAs (or miRs), many of which are known to be associated with cancer. To better understand the relationships between microRNAs and differ-ent cancers, we analyzed  9000 samples from 32 cancer types studies in The Cancer Genome Atlas (TCGA). Using the Thresher R package to per-form feature reduction, we found evidence for 21 biologically interpretable clusters of miRs. Many of these clusters were statistically associated with a specific type of cancer. Moreover, the clusters contain sufficient information to distinguish be-tween most types of cancer. We then used linear models to measure, genome-wide, how much vari-ation in gene expression could be explained by the 21 average expression values (“score”) of the clus-ters. Based on the  20,000 per-gene R2 values, we found that (a) mean differences between can-cer types explain about 40% of variation; (b) the 21 miR cluster scores explain about 30% of varia-tion, and (c) combining cancer type with the miR scores explained about 56% of the total genome-wide variation in gene expression. Our analysis of poorly explained genes shows that they are en-riched for olfactory receptor processes, sensory perception and nervous system processing which are necessary to receive and interpret signals from outside the organism. Therefore, it is reasonable for those genes to be always active and not get down-regulated by miRs. In contrast, highly ex-plained genes are characterized by genes trans-lating to proteins necessary for transport, plasma membrane, or metabolic processes that are heav-ily regulated processes inside the cell. The dis-tribution of R2 values suggests that other genetic regulatory elements such as transcription factors (TF) and methylation would help explain some of the remaining variation in gene expression. By building a combined microRNA-TF-methylation model, we can potentially predict the majority of human transcriptomic expression.
    </div>
	
</li></ol>

  <h3  id="2019" class="pubyear">2019</h3>
  <ol class="bibliography"><li><span id="aans19"><b>Asiaee*, A.</b>, Abrams*, Z. B., Nakayiza, S., Sampath, D., &amp; Coombes, K. R. (2019). Explaining Gene Expression Using Twenty-One MicroRNAs. <i>Computational Biology Workshop at ICML 2019</i>.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_aans19-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_aans19-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/aans19.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	
	

	<div class="dropDownBibtex" id="aans19-bibtex">
    <pre>@inproceedings{aans19,
  title = {Explaining Gene Expression Using Twenty-One MicroRNAs},
  author = {Asiaee*, Amir and Abrams*, Zachary B. and Nakayiza, Samantha and Sampath, Deepa and Coombes, Kevin R.},
  booktitle = {Computational Biology Workshop at ICML 2019},
  year = {2019}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="aans19-abstract">
    The transcriptome, or gene expression profile, of a tumor contains detailed information about the disease. Although advances in sequencing tech-nologies have generated larger and more infor-mative data sets, there are still many questions about how the transcriptome is regulated. One class of regulatory elements consists of micro-RNAs (or miRs), many of which are known to be associated with cancer. To better understand the relationships between microRNAs and differ-ent cancers, we analyzed ∼9000 samples from 32 cancer types studies in The Cancer Genome Atlas (TCGA). Using the Thresher R package to per-form feature reduction, we found evidence for 21 biologically interpretable clusters of miRs. Many of these clusters were statistically associated with a specific type of cancer. Moreover, the clusters contain sufficient information to distinguish be-tween most types of cancer. We then used linear models to measure, genome-wide, how much vari-ation in gene expression could be explained by the 21 average expression values (“score”) of the clus-ters. Based on the  20,000 per-gene R2 values, we found that (a) mean differences between can-cer types explain about 40% of variation; (b) the 21 miR cluster scores explain about 30% of varia-tion, and (c) combining cancer type with the miR scores explained about 56% of the total genome-wide variation in gene expression. Our analysis of poorly explained genes shows that they are en-riched for olfactory receptor processes, sensory perception and nervous system processing which are necessary to receive and interpret signals from outside the organism. Therefore, it is reasonable for those genes to be always active and not get down-regulated by miRs. In contrast, highly ex-plained genes are characterized by genes trans-lating to proteins necessary for transport, plasma membrane, or metabolic processes that are heav-ily regulated processes inside the cell. The dis-tribution of R2 values suggests that other genetic regulatory elements such as transcription factors (TF) and methylation would help explain some of the remaining variation in gene expression. By building a combined microRNA-TF-methylation model, we can potentially predict the majority of human transcriptomic expression.
    </div>
	
</li>
<li><span id="aocb19"><b>Asiaee, A.</b>, Oymak, S., Coombes, K. R., &amp; Banerjee, A. (2019). Data Enrichment: Multi-task Learning in High Dimension with Theoretical Guarantees. <i>Adaptive and Multi-Task Learning Workshop at ICML 2019</i>.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_aocb19-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_aocb19-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/aocb19.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	
	

	<div class="dropDownBibtex" id="aocb19-bibtex">
    <pre>@inproceedings{aocb19,
  title = {Data Enrichment: Multi-task Learning in High Dimension with Theoretical Guarantees},
  author = {Asiaee, Amir and Oymak, Samet and Coombes, Kevin R. and Banerjee, Arindam},
  booktitle = {Adaptive and Multi-Task Learning Workshop at ICML 2019},
  year = {2019}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="aocb19-abstract">
    Given samples from a group of related regres-sion tasks, a data-enriched model describes ob-servations by a common and per-group individual parameters. In high-dimensional regime, each pa-rameter has its own structure such as sparsity or group sparsity. In this paper, we consider the gen-eral form of data enrichment where data comes in a fixed but arbitrary number of tasks G and any convex function, e.g., norm, can character-ize the structure of both common and individual parameters. We propose an estimator for the high-dimensional data enriched model and investigate its statistical properties. We delineate the sam-ple complexity of our estimator and provide high probability non-asymptotic bound for estimation error of all parameters under a condition weaker than the state-of-the-art. We propose an itera-tive estimation algorithm with a geometric con-vergence rate. Overall, we present a first through statistical and computational analysis of inference in the data enriched model.
    </div>
	
</li>
<li><span id="cak18">Cho, M. H., <b>Asiaee*, A.</b>, &amp; Kurtek, S. (2019). Elastic Statistical Shape Analysis of Biological Structures with
             Case Studies: A Tutorial. <i>Bulletin of Mathematical Biology</i>, <i>81</i>(7), 2052–2073.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_cak18-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_cak18-bibtex" style="outline:none;">BibTeX</button>
    
    
	
	
	
	
	<a href="https://dx.doi.org/10.1007/s11538-019-00609-w"><button class="btn btnId btnPub--supplement" style="outline:none; position:relative;white-space: normal;">URL</button></a>
    

	<div class="dropDownBibtex" id="cak18-bibtex">
    <pre>@article{cak18,
  title = {Elastic Statistical Shape Analysis of Biological Structures with
               Case Studies: A Tutorial},
  author = {Cho, Min Ho* and Asiaee*, Amir and Kurtek, Sebastian},
  journal = {Bulletin of mathematical biology},
  volume = {81},
  number = {7},
  pages = {2052--2073},
  year = {2019},
  note = {https://dx.doi.org/10.1007/s11538-019-00609-w}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="cak18-abstract">
     We describe a recent framework for statistical shape analysis of curves and show its applicability to various biological datasets. The presented methods are based on a functional representation of shape called the square-root velocity function and a closely related elastic metric. The main benefit of this approach is its invariance to reparameterization (in addition to the standard shape-preserving transformations of translation, rotation and scale), and ability to compute optimal registrations (point correspondences) across objects. Building upon the defined distance between shapes, we additionally describe tools for computing sample statistics including the mean and covariance. Based on the covariance structure, one can also explore variability in shape samples via principal component analysis. Finally, the estimated mean and covariance can be used to define Wrapped Gaussian models on the shape space, which are easy to sample from. We present multiple case studies on various biological datasets including (1) leaf outlines, (2) internal carotid arteries, (3) Diffusion Tensor Magnetic Resonance Imaging fiber tracts, (4) Glioblastoma Multiforme tumors, and (5) vertebrae in mice. We additionally provide a MATLAB package that can be used to produce the results given in this manuscript.
    </div>
	
</li></ol>

  <h3  id="2018" class="pubyear">2018</h3>
  <ol class="bibliography"><li><span id="azwa18">Abrams, Z. B., Zucker, M., Wang, M., <b>Asiaee Taheri, A.</b>, Abruzzo, L. V., &amp; Coombes, K. R. (2018). Thirty biologically interpretable clusters of transcription
              factors distinguish cancer type. <i>BMC Genomics</i>, <i>19</i>(1), 738.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_azwa18-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_azwa18-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/azwa18.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	
	
	<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5093-z"><button class="btn btnId btnPub--supplement" style="outline:none; position:relative;white-space: normal;">URL</button></a>
    

	<div class="dropDownBibtex" id="azwa18-bibtex">
    <pre>@article{azwa18,
  title = {Thirty biologically interpretable clusters of transcription
                factors distinguish cancer type},
  author = {Abrams, Zachary B and Zucker, Mark and Wang, Min and Asiaee Taheri, Amir and Abruzzo, Lynne V and Coombes, Kevin R},
  journal = {BMC genomics},
  volume = {19},
  number = {1},
  pages = {738},
  note = {https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-5093-z},
  year = {2018}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="azwa18-abstract">
    BACKGROUND: Transcription factors are essential regulators of gene expression and play critical roles in development, differentiation, and in many cancers. To carry out their regulatory programs, they must cooperate in networks and bind simultaneously to sites in promoter or enhancer regions of genes. We hypothesize that the mRNA co-expression patterns of transcription factors can be used both to learn how they cooperate in networks and to distinguish between cancer types. RESULTS: We recently developed a new algorithm, Thresher, that combines principal component analysis, outlier filtering, and von Mises-Fisher mixture models to cluster genes (in this case, transcription factors) based on expression, determining the optimal number of clusters in the process. We applied Thresher to the RNA-Seq expression data of 486 transcription factors from more than 10,000 samples of 33 kinds of cancer studied in The Cancer Genome Atlas (TCGA). We found that 30 clusters of transcription factors from a 29-dimensional principal component space were able to distinguish between most cancer types, and could separate tumor samples from normal controls. Moreover, each cluster of transcription factors could be either (i) linked to a tissue-specific expression pattern or (ii) associated with a fundamental biological process such as cell cycle, angiogenesis, apoptosis, or cytoskeleton. Clusters of the second type were more likely also to be associated with embryonically lethal mouse phenotypes. CONCLUSIONS: Using our approach, we have shown that the mRNA expression patterns of transcription factors contain most of the information needed to distinguish different cancer types. The Thresher method is capable of discovering biologically interpretable clusters of genes. It can potentially be applied to other gene sets, such as signaling pathways, to decompose them into simpler, yet biologically meaningful, components.
    </div>
	
</li>
<li><span id="aggy18"><b>Asiaee T., A.</b>, Goel, H., Gosh, S., Yegneswaran, V., &amp; Banerjee, A. (2018). Time Series Deinterleaving of DNS Traffic. <i>1st Deep Learning and Security Workshop (DLS)</i>.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_aggy18-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_aggy18-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/aggy18.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	
	

	<div class="dropDownBibtex" id="aggy18-bibtex">
    <pre>@inproceedings{aggy18,
  title = {Time Series Deinterleaving of DNS Traffic},
  author = {Asiaee T., Amir and Goel, Hardik and Gosh, Shalini and Yegneswaran, Vinod and Banerjee, Arindam},
  booktitle = {1st Deep Learning and Security Workshop (DLS)},
  year = {2018}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="aggy18-abstract">
    Stream deinterleaving is an important problem with various applications in the cybersecurity domain.  In this paper, we consider the specific problem of deinterleaving DNS data streams using machine-learning techniques, with the objective of automating the extraction of malware domain sequences.  We first develop a generative model for user request generation and DNS stream interleaving.  Based on these we evaluate various inference strategies for deinterleaving including augmented HMMs and LSTMs on synthetic datasets.  Our results demonstrate that state-of-the-art LSTMs outperform more traditional augmented HMMs in this application domain.
    </div>
	
</li></ol>

  <h3  id="2016" class="pubyear">2016</h3>
  <ol class="bibliography"><li><span id="ascb16"><b>Asiaee T., A.</b>, Chaterjee, S., &amp; Banerjee, A. (2016). High Dimensional Structured Estimation with Noisy Designs. <i>16th SIAM International Conference on Data Mining (SDM)</i>, 801–809.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_ascb16-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_ascb16-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/ascb16.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	
	

	<div class="dropDownBibtex" id="ascb16-bibtex">
    <pre>@inproceedings{ascb16,
  title = {High Dimensional Structured Estimation with Noisy Designs},
  author = {Asiaee T., Amir and Chaterjee, Soumyadeep and Banerjee, Arindam},
  booktitle = {16th SIAM International Conference on Data Mining (SDM)},
  pages = {801--809},
  year = {2016},
  organization = {SIAM}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="ascb16-abstract">
    Structured estimation methods, such as LASSO, have received considerable attention in recent years and substantial progress has been made in extending such methods to general norms and non-Gaussian design matrices. In real world problems, however, covariates are usually corrupted with noise and there have been efforts to generalize structured estimation method for noisy covariate setting. In this paper we first show that without any information about the noise in covariates, currently established techniques of bounding statistical error of estimation fail to provide consistency guarantees. However, when information about noise covariance is available or can be estimated, then we prove consistency guarantees for any norm regularizer, which is a more general result than the state of the art. Next, we investigate empirical performance of structured estimation, specifically LASSO, when covariates are noisy and empirically show that LASSO is not consistent or stable in the presence of additive noise. However, prediction performance improves quite substantially when the noise covariance is available for incorporating in the estimator.
    </div>
	
</li></ol>

  <h3  id="2015" class="pubyear">2015</h3>
  <ol class="bibliography"><li><span id="gabz15">Golnari*, G., <b>Asiaee T.*, A.</b>, Banerjee, A., &amp; Zhang, Z.-L. (2015). Revisiting Non-Progressive Influence Models: Scalable Influence Maximization in Social Networks. <i>31st Conference on Uncertainty in Artificial Intelligence (UAI)</i>, 316–325.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_gabz15-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_gabz15-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/gabz15.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	<a href="/publications/gabz15.sup.pdf"><button class="btn btnId btnPub--supplement" style="outline:none; position:relative;white-space: normal;">Supplement</button></a>
    
	
	

	<div class="dropDownBibtex" id="gabz15-bibtex">
    <pre>@inproceedings{gabz15,
  title = {Revisiting Non-Progressive Influence Models: Scalable Influence Maximization in Social Networks.},
  author = {Golnari*, Golshan and Asiaee T.*, Amir and Banerjee, Arindam and Zhang, Zhi-Li},
  booktitle = {31st Conference on Uncertainty in Artificial Intelligence (UAI)},
  pages = {316--325},
  year = {2015}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="gabz15-abstract">
    Influence maximization in social networks has been studied extensively in computer science community for the last decade. However, almost all of the efforts have been focused on the <i>progressive</i> influence models, such as independent cascade (IC) and Linear threshold (LT) models, which cannot capture the \textitreversibility of choices. In this paper, we present the Heat Conduction (HC) model which is a \textitnon-progressive influence model and has favorable real-world interpretations. Moreover, we show that HC unifies, generalizes, and extends the existing non-progressive models, such as the Voter model \citeeven-dar_note_2007 and non-progressive LT \citekempe_maximizing_2003. We  then prove that selecting the optimal seed set of influential nodes is NP-hard for HC but by establishing the submodularity of influence spread, we can tackle the influence maximization problem with a scalable and provably near-optimal greedy algorithm. To  the  best  of  our  knowledge, we  are  the  first  to present a scalable solution for influence maximization under non-progressive LT model, as a special case of HC model. In sharp contrast to the other greedy influence maximization methods, our fast and efficient <font style="font-variant: small-caps">C2Greedy</font> algorithm benefits from two analytically computable steps: <i>closed-form</i> computation for finding the influence spread as well as the greedy seed selection. Through extensive experiments on several and large real and synthetic networks, we show that <font style="font-variant: small-caps">C2Greedy</font> outperforms the state-of-the-art methods, under HC model, in terms of both influence spread and scalability. 
    </div>
	
</li></ol>

  <h3  id="2013" class="pubyear">2013</h3>
  <ol class="bibliography"><li><span id="asaa13"><b>Asiaee T., A.</b>, Afshar, M., &amp; Asadpour, M. (2013). Influence maximization for informed agents in collective behavior. In <i>Distributed Autonomous Robotic Systems</i> (pp. 389–402). Springer.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_asaa13-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_asaa13-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/asaa13.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	
	

	<div class="dropDownBibtex" id="asaa13-bibtex">
    <pre>@incollection{asaa13,
  title = {Influence maximization for informed agents in collective behavior},
  author = {Asiaee T., Amir and Afshar, Mohammad and Asadpour, Masoud},
  booktitle = {Distributed Autonomous Robotic Systems},
  pages = {389--402},
  year = {2013},
  publisher = {Springer}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="asaa13-abstract">
    Control of collective behavior is an active topic in biology, social, and computer science. In this work we investigate how a minority of informed agents can influence and control the whole society through local interactions. The problem we specifically target is that a minority of people with a bounded budget for initiating new social relations attempt to control the collective behavior of a society and move the crowd toward a specific goal. Assuming that local interactions can only take place between friends, the minority has to initiate some new relations with the majority. The total cost of new relations is limited to a budget. The problem is then finding the optimal links in order to gain maximum impact on the society. We will model the problem as a diffusion process in a social network. The proof of NP-hardness of the problem for Local Interaction Game model of diffusion is presented. Simulations show that the proposed method surpasses the popular strategies based on degree and distance centrality in performance.
    </div>
	
</li></ol>

  <h3  id="2012" class="pubyear">2012</h3>
  <ol class="bibliography"><li><span id="atbs12"><b>Asiaee T., A.</b>, Tepper, M., Banerjee, A., &amp; Sapiro, G. (2012). If you are happy and you know it... tweet. <i>21st ACM International Conference on Information and Knowledge Management (CIKM)</i>, 1602–1606.</span>


    
    <button class="btn btnId btnPub--abstract" id="b_atbs12-abstract" style="outline:none;">Abstract</button>
    

	<button class="btn btnId btnPub--BibTex" id="b_atbs12-bibtex" style="outline:none;">BibTeX</button>
    
    
	<a href="/publications/atbs12.pdf"><button class="btn btnId btnPub--download" style="outline:none; position:relative;white-space: normal;">PDF</button></a>
    
	
	
	
	

	<div class="dropDownBibtex" id="atbs12-bibtex">
    <pre>@inproceedings{atbs12,
  title = {If you are happy and you know it... tweet},
  author = {Asiaee T., Amir and Tepper, Mariano and Banerjee, Arindam and Sapiro, Guillermo},
  booktitle = {21st ACM international conference on Information and knowledge management (CIKM)},
  pages = {1602--1606},
  year = {2012},
  organization = {ACM}
}
</pre>
    </div>
	
	
	<div class="dropDownAbstract" id="atbs12-abstract">
    Extracting sentiment from Twitter data is one of the fundamental problems in social media analytics. Twitter’s length constraint renders determining the positive/negative sentiment of a tweet difficult, even for a human judge. In this work we present a general framework for per-tweet (in contrast with batches of tweets) sentiment analysis which consists of: (1) extracting tweets about a desired target subject, (2) separating tweets with sentiment, and (3) setting apart positive from negative tweets. For each step, we study the performance of a number of classical and new machine learning algorithms. We also show that the intrinsic sparsity of tweets allows performing classification in a low dimensional space, via random projections, without losing accuracy. In addition, we present weighted variants of all employed algorithms, exploiting the available labeling uncertainty, which further improve classification accuracy. Finally, we show that spatially aggregating our per-tweet classification results produces a very satisfactory outcome, making our approach a good candidate for batch tweet sentiment analysis.
    </div>
	
</li></ol>




        
      </section>

      <footer class="page__meta">
        
        


        
      </footer>

      

      
    </div>

    
  </article>

  
  
</div>

    </div>

    
      <div class="search-content">
        <div class="search-content__inner-wrap">
  <input type="text" id="search" class="search-input" tabindex="-1" placeholder="Enter your search term..." />
  <div id="results" class="results"></div>
</div>
      </div>
    

    <div class="page__footer">
      <footer>
        <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
        <div class="page__footer-follow">
  <ul class="social-icons">
    
      <li><strong>Follow:</strong></li>
    
    
      <li><a href="https://twitter.com/aasiaeet"><i class="fa fa-fw fa-twitter-square" aria-hidden="true"></i> Twitter</a></li>
    
    
    
    
    
    <li><a href="https://www.amirasiaee.com/feed.xml"><i class="fa fa-fw fa-rss-square" aria-hidden="true"></i> Feed</a></li>
  </ul>
</div>

<div class="page__footer-copyright">&copy; 2022 Amir Asiaee. Powered by <a href="https://jekyllrb.com" rel="nofollow">Jekyll</a> &amp; <a href="https://mademistakes.com/work/minimal-mistakes-jekyll-theme/" rel="nofollow">Minimal Mistakes</a>.</div>

<!-- AJax for DropDown Effect of Abstract and BibTex -->
<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
<script>
$(document).ready(function(){
	var str =$(this).attr('id');
	
    $(".btnId").click(function(){
        var str = $(this).attr('id');
		var ret = str.split("_");
		var id = ret[1];
		$('#' + id).toggle();
    });
});
</script>

<!-- New to convert MathJax to KaTex 
<script type="text/javascript">
$("script[type='math/tex']").replaceWith(
  function(){
    var tex = $(this).text();
    return "<span class=\"inline-equation\">" + 
           katex.renderToString(tex) +
           "</span>";
});

$("script[type='math/tex; mode=display']").replaceWith(
  function(){
    var tex = $(this).text();
    return "<div class=\"equation\">" + 
           katex.renderToString("\\displaystyle "+tex) +
           "</div>";
});
</script>
-->

<!-- Previous for changing MathJax to KaTex -->
<script>
  $("script[type='math/tex']").replaceWith(function() {
      var tex = $(this).text();
      return katex.renderToString(tex.replace(/%.*/g, ''), {displayMode: false});
  });

  $("script[type='math/tex; mode=display']").replaceWith(function() {
      var tex = $(this).html();
      return katex.renderToString(tex.replace(/%.*/g, ''), {displayMode: true});
  });
</script>

      </footer>
    </div>

    
  <script src="https://www.amirasiaee.com/assets/js/main.min.js"></script>



  <script src="https://www.amirasiaee.com/assets/js/lunr.min.js"></script>
  <script src="https://www.amirasiaee.com/assets/js/lunr-en.js"></script>





  <script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-81365128-3']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>






  </body>
</html>